### PR TITLE
[lazy] fix lazy cls init

### PR DESCRIPTION
**Pull request by flybird11111**
## 📌 Checklist before creating the PR

- [ ] I have created an issue for this PR for traceability
- [x] The title follows the standard format: `[doc/gemini/tensor/...]: A concise description`
- [ ] I have added relevant tags if possible for us to better distinguish different PRs
- [x] I have installed pre-commit: `pip install pre-commit && pre-commit install`

## 🚨 Issue number

> Link this PR to your issue with words like fixed to automatically close the linked issue upon merge
>
> e.g. `fixed #1234`, `closed #1234`, `resolved #1234`

## 📝 What does this PR do?

> Summarize your work here.
> if you have any plots/diagrams/screenshots/tables, please attach them here.

fix lazy cls init

## 💥 Checklist before requesting a review

- [ ] I have linked my PR to an issue ([instruction](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue))
- [ ] My issue clearly describes the problem/feature/proposal, with diagrams/charts/table/code if possible
- [x] I have performed a self-review of my code
- [ ] I have added thorough tests.
- [ ] I have added docstrings for all the functions/methods I implemented

## ⭐️ Do you enjoy contributing to Colossal-AI?

- [ ] 🌝 Yes, I do.
- [ ] 🌚 No, I don't.

Tell us more if you don't enjoy contributing to Colossal-AI.

### CODE REVIEW

**Review by ver217 — APPROVED** [review]

File: .github/workflows/build_on_pr.yml [hunk 1]
```diff
@@ -140,7 +140,6 @@ jobs:
 
       - name: Install Colossal-AI
         run: |
-          BUILD_EXT=1 pip install -v -e .
```
Comment (ver217) [inline .github/workflows/build_on_pr.yml:143]: This should be `pip install -v -e .`

File: requirements/requirements-test.txt [hunk 2]
```diff
@@ -20,3 +20,4 @@ pydantic
 ray
 peft>=0.7.1
 #auto-gptq now not support torch1.12
+galore_torch
```
Comment (ver217) [inline requirements/requirements-test.txt:23]: This is written in `requirements.txt` and can be removed here.

**Review by ver217 — CHANGES_REQUESTED** [review]

File: requirements/requirements-test.txt (hunk 2 above)
Comment (flybird11111) [reply requirements/requirements-test.txt]: updated.

File: .github/workflows/build_on_pr.yml (hunk 1 above)
Comment (flybird11111) [reply .github/workflows/build_on_pr.yml:143]: updated

**Review by ver217 — APPROVED** [review]

**Pull request merged**